README.org

```diff
@@ -7,8 +7,7 @@ Also, please do consider contributing back to the Emacs community. One major way
 
 Above, all enjoy using Emacs. The community, that more than anything, makes Emacs a great ecosystem and place to develop, learn and grow.
 
-#+BEGIN_QUOTE
```

Comment:
Sorry, I place the TOC within a QUOTE block, which is just an... aesthetic preference. It looks good in GitHub's Org->HTML renderer
![screenshot_20180925_232709](https://user-images.githubusercontent.com/1370070/46025230-0bbccb00-c11b-11e8-8704-9cbc35988336.png)